packages/cashier/src/Stores/Cashier/cashier-store.js

```diff
@@ -232,11 +232,12 @@ export default class CashierStore extends BaseStore {
                 this.root_store.client.currency,
             ],
             async () => {
-                // wait for get_settings so is_virtual gets populated in client-store
+                // wait for client settings to be populated in client-store
                 await this.WS.wait('get_settings');
 
-                if (this.root_store.client.is_logged_in && !this.root_store.client.is_virtual) {
+                if (this.root_store.client.is_logged_in) {
```

Comment:
why are we removing `!is_virtual` condition?

Reply:
Currently, there is an inconsistency in showing DP2P option as when you reload page on virtual account, it won't be shown, while if you open a real account and then switch to virtual, it will be shown. By removing this condition, I execute below logic no matter what account user has when mounted, so DP2P option will always be there.

Reply:
`checkP2pStatus` request for  `p2p_advertiser_info` api. If it is a virtual account, we can avoid calling `p2p_advertiser_info` api.

https://github.com/binary-com/deriv-app/pull/2560/files
this might fix the problem you mentioned. can you please check?

Reply:
Thanks Bala, you're right, this would just make an unnecessary call. I added the condition back.